FBSnapshotTestCaseTests/FBSnapshotControllerTests.m

```diff
@@ -194,8 +193,8 @@ - (void)testCompareReferenceImageWithLowPixelToleranceShouldMatch
 
 - (UIImage *)_bundledImageNamed:(NSString *)name type:(NSString *)type
 {
-    NSBundle *bundle = [NSBundle bundleForClass:[self class]];
-    NSString *path = [bundle pathForResource:name ofType:type];
+    NSURL *url = [[NSURL fileURLWithPath: @__FILE__] URLByDeletingLastPathComponent];
+    NSString *path = [[url URLByAppendingPathComponent: [NSString stringWithFormat:@"%@.%@", name, type]] path];
```

Comment:
I'm assuming this is because the image isn't visible in the bundle? Is there a better way to do this because this seems like a hack that finds the temp directory storing the _code file_ and using that path to find the image. Was `bundle` nil before?

Reply:
Yes, today it's not possible load bundle resource in the SPM, but has open issue in `swift .org` about this. I' created this hack as temp solution, if bundle fail we can see in the unit test

Reply:
Might be worth adding a comment in the code explaining that?

Reply:
I read file with using reserved key `__FILE__`, this create relative path from file test

Reply:
A comment in the code linking to the open issue would be suffice

Reply:
We can bundle resources in swift packages now https://developer.apple.com/documentation/swift_packages/bundling_resources_with_a_swift_package